library/errata_tool_product_version.py

```diff
@@ -1,15 +1,16 @@
+from email.policy import default
 from ansible.module_utils.basic import AnsibleModule
 from ansible.module_utils import common_errata_tool
 
 
 ANSIBLE_METADATA = {
-    'metadata_version': '1.0',
-    'status': ['preview'],
-    'supported_by': 'community'
+    "metadata_version": "1.0",
+    "status": ["preview"],
+    "supported_by": "community",
 }
 
 
-DOCUMENTATION = '''
+DOCUMENTATION = """
```

Comment:
Please read the earlier PR in https://github.com/ktdreyer/errata-tool-ansible/pull/264 . We need to have new documentation for any parameters we add.

Reply:
Hi I am using autoformater and did not know that would be a problem will fix that. 

Best Regards 
Stanislav Jochman